packages/rest/src/http-handler.ts

```diff
@@ -41,7 +41,7 @@ export class HttpHandler {
     this._routes.registerRoute(route);
   }
 
-  registerApiDefinitions(defs: DefinitionsObject) {
+  registerApiDefinitions(defs: SchemaObject) {
```

Comment:
Before this change, this method was accepting a key/value map of definitions. After your change, it accepts a single definition only. This is fine with me, but please change the method name from plural `registerApiDefinitions` to singular `registerApiDefinition`. Maybe we should come up with a better name, e.g. `registerApiSchema`?